homework27/src/ba/bitcamp/homework/tasks/Task1.java

```diff
@@ -0,0 +1,120 @@
+package ba.bitcamp.homework.tasks;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * This class is used for counting first letter from file using consumer
+ * producer metodology
+ * 
+ *
+ *
+ */
+public class Task1 {
+    //Declaring global variables
+	private static BufferedReader br = null;
+	private static Integer counter = 0;
+	private static LinkedBlockingQueue<Runnable> queue;
+	private static ArrayList<Producer> producers;
+	private static ArrayList<String> firstLetters;
+
+	public static void main(String[] args) {
+
+		queue = new LinkedBlockingQueue<Runnable>();
+		firstLetters = new ArrayList<String>();
+		try {
+			br = new BufferedReader(new FileReader(new File("file.txt")));
```

Comment:
BufferedReader instances must be closed.
Good approach would be to have 'finally' block with: `if (br != null) {br.close();}`